practice_app/pages/api/getUpdatedPrice.js

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios'
+
+
+export default (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET': {
+        	// This is not important for now
+        	var productID = query.pid 
+        	// Creating a random product price
+        	var productPrice = 60
+        	// hold client's ip adress
+        	var clientIP = query.ip == null ? "" : query.ip
+
+        	//use two apis synchroniously
+	        Promise.all([
+				axios.get("https://json.geoiplookup.io/"+clientIP),
+				axios.get("https://api.exchangeratesapi.io/latest")
+						])
+				.then(function (responses) {
+					// Get a JSON object from each of the responses
+					var takencurrencyType = responses[0].data.currency_code
```

Comment:
- Please use camelCase naming convention for variable names in JS so it should be `takenCurrencyType`
- Also please do not use `var` for variable declaration as it can cause problems and is considered old JS. Use `let` for mutable variables and `const` for immutables variables
- Please return an error to the client when you are not able to fetch the currency. Currently you are catching the error and logging it to the console on the server, so the client has no idea what happened.